rules/ast/functions.go

```diff
@@ -291,6 +291,32 @@ func dropCommonLabelsImpl(timestamp clientmodel.Timestamp, args []Node) interfac
 	return vector
 }
 
+// === round(vector VectorNode) Vector ===
+func roundImpl(timestamp clientmodel.Timestamp, args []Node) interface{} {
+	// round returns a number rounded to nearest integer
```

Comment:
Again, sentence-style comments please:

"round returns ... to nearest integer. Ties are resolved ...."

Otherwise :+1:, awesome!